src/scripts/main.js

```diff
@@ -1,8 +1,26 @@
 'use strict';
 
 const people = require('./lib/people');
+const table = document.querySelector('tbody');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+people.forEach(person => {
+  let gender;
```

Comment:
You can use a ternary operator to simplify the gender assignment. Replace lines 9-16 with 'const gender = person.sex === 'm' ? 'Male' : 'Female';'